Packs/Salesforce/Integrations/SalesforceV2/SalesforceV2.py

```
@@ -0,0 +1,1041 @@
+        propertiesA = ['Id', 'Name']
+        usersA = client.queryObjects(propertiesA, "Account", conditionA).get("records")
+    else:
+        return_error('You must specify a Case Number')
+    return objectToEntry(client, usersA)
+
+
+def post_case_comment_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    results = client.postCaseComment(args.get('public'), args.get(
+        'oid', None), args.get('caseNumber', None), args.get('text', None))
+    if results.get('success', None):
+        caseComment = client.getObject(f"{results.get('id')}")
+    else:
+        return_results(f"Unable to post case comment. Error Encountered was:{json.dumps(results.get('errors', None))}")
+
+    return objectToEntry(client, caseComment)
+
+
+def create_case_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    data = {
+        "Subject": args.get('subject'),
+        "Description": args.get('description'),
+        "Status": args.get('status'),
+        "Origin": args.get('origin'),
+        "Priority": args.get('priority'),
+        "Type": args.get('caseType')
+    }
+
+    results = client.createObject('Case', data)
+    if results.get('success', None):
+        case = client.getObject(f"{results.get('id')}")
+    else:
+        return_results(f"Unable to create case. Error Encountered was:{json.dumps(results.get('errors', None))}")
+
+    return objectToEntry(client, case)
+
+
+def update_case_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    if args.get('oid', None) and args.get('caseNumber', None):
+        return_error('You must specify object ID or a Case Number')
+
+    if args.get('caseNumber', None):
+        condition = f"CaseNumber='{args.get('caseNumber')}'"
+        cases = client.queryObjects(['Id'], 'Case', condition).get('records')
+        if len(cases) > 0:
+            oid = cases[0].get('Id')
+        else:
+            return_error('Unable to update case -> Invalid Case Number provided')
```

better to raise an exception here as well.